mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
No - we test for equality to check whether an option has changed from the default value or not. We need this in a myriad of places, for instance when we decide whether or not to write a changed option to file.